pkg/services/sqlstore/migrations/playlist_mig.go

```diff
@@ -1,13 +1,25 @@
 package migrations
 
-import . "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+import (
+	. "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
 
 func addPlaylistMigrations(mg *Migrator) {
+	playlistV2 := Table{
```

Comment:
this may look like a big change, but is really just defining the schema once and passing it to each migration rather than creating a new one for each migration.